src/Delivery/ResourceBuilder.php

```diff
@@ -169,14 +169,17 @@ private function buildArray(array $data, array $rawDataList = null)
      */
     private function buildAsset(array $data)
     {
+        $sys = $this->buildSystemProperties($data['sys']);
+        $locale = $sys->getLocale();
+
         $fields = $data['fields'];
-        $files = array_map([$this, 'buildFile'], $fields['file']);
+        $files = array_map([$this, 'buildFile'], $this->normalizeFieldData($fields['file'], $locale));
 
         $asset = new Asset(
-            isset($fields['title']) ? $fields['title'] : null,
-            isset($fields['description']) ? $fields['description'] : null,
+            isset($fields['title']) ? $this->normalizeFieldData($fields['title'], $locale) : null,
+            isset($fields['description']) ? $this->normalizeFieldData($fields['description'], $locale) : null,
             $files,
-            $this->buildSystemProperties($data['sys'])
+            $sys
```

Comment:
Both work. See this part: https://github.com/contentful/contentful.php/pull/48/files/66bf3c2c23eaf022ccd2ef12c9e07d221ad7713f#diff-52781e0c24eff4092e33103bf162cbd4R179